web-console/knikubevirt/Create-vm/2.networking/create-vm-networking.md

```diff
@@ -0,0 +1,19 @@
+# Create A Virtual Machine
+
+## Step 2: Networking
+
+![Default](img/3-00.png)
```

Comment:
We should hide the breadcrumbs here since users should really take the "cancel" action to get back. We don't show breadcrumbs on other forms in the console today because we don't want a user losing their form inputs by accident. 

@lizsurette @serenamarie125 In general I think we should decide if having the project dropdown makes sense for forms. We are inconsistent today but sounded like the latest discussions concluded that we should not show the secondary project bar for forms and instead if it is a project-scoped resource, add the project selection as a field in the form with the choice defaulting to the current project. This seems more clear to users so folks don't miss the project bar. We can work on this in the design conventions, maybe not critical to cover in this PR :)

Reply:
The way I see it, pretty much all the navigation and actions wrapping the form are exit point from that form. clicking the breadcrumb seems, to me, equivalent to clicking any other item on the top or side menu (and essentially to some users equivalent to "Cancel"). Once we use an In-page, the only thing we can do is add a "discard/exit/cancel" confirmation modal (and "save as draft" in the future). The breadcrumbs are still a valid orientation help and a way to return to the previous page since the cancel button may be at the bottom of a long scroll.
Does that make sense?

Reply:
I see your point @beanh66 about the breadcrumbs, I think we need to be consistent. I added them in the first pass of the in page wizard designs b/c I wanted to add more clarity for where the user was, this page doesn't exist in the nav so via the breadcrumbs they can see where they are. I agree with @Ranelim point about clicking away via a nav item would be the same experience as clicking away via the breadcrumb. In either case we would alert the user that data would be lost if they added any info in the form. This is a discussion that impacts the entire console though so I think it's worth figuring out. WDYT @beanh66 I'll leave it up to you since you've been apart of this conversation longer than we have been :)

Reply:
I opened a ['Breadcrumbs on in-page wizard' issue](https://github.com/openshift/openshift-origin-design/issues/244) so it won't block this PR in case this goes to a longer discussion.

Reply:
Thanks @Ranelim that makes sense to me.

Reply:
Thanks @Ranelim that sounds good to me.

Reply:
Removed the breadcrumbs from this design for now